admin/proc-disabling-telemetry.adoc

```diff
@@ -0,0 +1,66 @@
+[id='disabling-telemetry-data-collection_{context}']
+= Disabling telemetry data collection in {product-very-short}
+
+To disable telemetry data collection, you must configure the `analytics-provider-segment` plugin either using the Helm Chart or the {product} Operator. 
+
+[NOTE]
+====
+If the `analytics-provider-segment` plugin is already present in your dynamic plugins configuration, set the value of the `plugins.disabled` parameter to `true` to disable telemetry data collection.
+====
+
+.Prerequisites
```

Comment:
Lets remove this whole section. Those prereqs doesn't make much sense. This is part of the admin documentation, from that it should be clear that this is for RHDH administrators.